### PR TITLE
Media lib front fixes

### PR DESCRIPTION
**Pull request by virginieky**
#### Description of what you did:

- Videos "play" icons alignment
- Display permanently the "play" icon on the edit single card
- EditModal informations box border-radius
- Center the "Crop the original" / "Duplicate and crop" buttons
- Fix all alert boxes 
- Remove  "x assets selected" in the CM modal behind the browse tab title
- Enable to edit the files from the "selected" tab of the CM modal 
- Disable search autocomplete
- CTM input media "play" icon on the video thumbnail
- Pagination alignment
- Fix CTM listview extension
- Fix edit icon displayed on file checked
- CurrentTab state to keep selected tab through modal navigation
- Change the behavior of the "select only one" so that selecting another asset automatically unselect the previous one
- Use the "Small" format in the CM input file preview rather than the thumbnail if not, fall back on the URL.
- Modal list scrollbar display on FF

### CODE REVIEW

File: packages/strapi-plugin-upload/admin/src/components/SelectedAssets/SortableList.js [hunk 1]
```diff
@@ -6,11 +6,21 @@ import { prefixFileUrlWithBackendUrl } from 'strapi-helper-plugin';
...
+  moveAsset,
+  noNavigation,
+  onChange,
+  onClickEditFile,
+  selectedItems,
+}) => {
   const handleClick = e => {
     e.stopPropagation();
   };
+  console.log(noNavigation);
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/components/SelectedAssets/SortableList.js:23]: Can you remove it?

File: packages/strapi-plugin-upload/admin/src/components/BrowseAssets/index.js [hunk 2]
```diff
@@ -132,22 +143,23 @@ const BrowseAssets = () => {
...
-              params={paginationParams}
-            />
+            <Padded left right size="xs">
+              <PageFooter
+                context={{ emitEvent: () => {} }}
+                count={count}
+                onChangeParams={handleChangeListParams}
+                params={paginationParams}
+              />
+            </Padded>
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/components/BrowseAssets/index.js:162]: The pagination does not work anymore in the modal. I don't know why.

File: packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js [hunk 3]
```diff
@@ -36,13 +34,13 @@ function MediaPreviewList({ hoverable, files }) {
 
   const renderFile = file => {
     const { mime, name } = file;
-    const fileType = includes(mime, 'image') ? 'image' : getFileType(name);
+    const fileExtension = getFileExtension(mime);
 
     return (
       <MediaPreviewFile className={hoverable ? 'hoverable' : ''}>
-        {fileType ? (
+        {fileExtension ? (
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js:41]: No not always in fact if you open 2 tabs select a file in the media-input from the first tab and delete the exact same file from the second tab you will still able to save your entry even though the selected file does not exist anymore. Thus, when you receive your data in the list view you will get an empty object for the attribute. This condition prevents the admin from breaking.

[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js:41]: Yes, I agree with this case. But here, technically, `fileExtension` is always defined as `getFileExtension` return a string for all cases.

getFileExtension.js
```js
const getFileExtension = mime => (mime ? mime.split(/[\s/]+/)[1] : 'undefined');
```

[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js:41]: @soupette Yes, I agree with this case. But here, technically, `fileExtension` is always defined as `getFileExtension` return a string for all cases.

getFileExtension.js
```js
const getFileExtension = mime => (mime ? mime.split(/[\s/]+/)[1] : 'undefined');
```

[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js:41]: Sure you are right it should return  `null` like it was previously

File: packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+describe('HELPER PLUGIN | utils | getFileExtension', () => {
+  it('should return undefined if ext does not exits', () => {
+    const ext = null;
+    const expected = null;
+
+    expect(getFileExtension(ext)).toEqual(expected);
+  });
+
+  it('should return png if ext string is image/png', () => {
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js:11]: Can you change the test title here ext is .png not image/png

File: packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    const ext = '.png';
+    const expected = 'png';
+
+    expect(getFileExtension(ext)).toEqual(expected);
+  });
+
+  it('should return mp4 if ext string is video/mp4', () => {
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js:18]: same

File: packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    const ext = '.mp4';
+    const expected = 'mp4';
+
+    expect(getFileExtension(ext)).toEqual(expected);
+  });
+
+  it('should return html if ext string is text/html', () => {
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js:25]: same

File: packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    const ext = '.bin';
+    const expected = 'bin';
+
+    expect(getFileExtension(ext)).toEqual(expected);
+  });
+
+  it('should return pdf if ext string is application/pdf', () => {
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/utils/tests/getFileExtension.test.js:32]: same

File: packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/index.js [hunk 8]
```diff
@@ -296,7 +305,7 @@ const InputModalStepperProvider = ({
 
   const getFilters = () => {
     const compactedParams = compactParams(params);
-    const searchParams = generateSearchFromFilters(compactedParams, ['_limit', '_sort', '_start']);
+    const searchParams = generateSearchFromFilters(compactedParams);
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/index.js:308]: You should add the filters to remove to the `getFilters` and pass the option in the `fetchMediaLibFilesCount`

```
const fetchMediaLibFilesCount = () => {
  const paramsToSend = getFilters(['_limit', '_sort', '_start']);
}

const getFilters = (filtersToOmit = []) => {
  // ...
  const searchParams = generateSearchFromFilters(compactedParams, filtersToOmit);
}
```

**Pull request merged**